internal/biz/greeter.go

```diff
@@ -0,0 +1,34 @@
+package biz
+
+import (
+	"context"
+	"demo/internal/data/models"
+	"demo/internal/data/repositories/impl"
+
+	v1 "demo/api/helloworld/v1"
```

Comment:
pbTypes (protobuf) or something else would be a better alias than v1.

Reply:
the alias is auto generated by goland